### ISSUE TITLE
Validation script

### ISSUE THREAD

**Issue by kirilklein** [FGA-DIKU/EHR#93]
Write a script that evaluates the model on a test set.  This would come in handy and would allow out of time evaluation.

**Comment by kirilklein** [comment]
Instead of a validation script, we introduce a boolean parameter test which can be turned of for hyperparameter tuning and turned on for final model training.
